example-clients/ipload.c

```diff
@@ -45,17 +51,33 @@ signal_handler (int sig)
 		fprintf (stderr, "(failed), status = 0x%2.0x\n", status);
 	else
 		fprintf (stderr, "(succeeded)\n");
+	if (autoclose_opt)
+		jack_deactivate(client);
 	jack_client_close (client);
 	exit (0);
 }
 
 static void
+registration_callback (const char *name, int reg, void *arg)
+{
+	if (reg || strcmp(intclient_name, name))
+		return;
+
+	/* this will stop the wait loop and thus close this application. */
+	idling = 0;
+
+	/* unused */
+	return; (void)arg;
```

Comment:
I am unsure here. Casting to void already is already the signal to the compiler, that  arg is unused so maybe no comment needed? Also if I as a reader did not know that `/* unused*/` refers to `arg` and not the return, I would still wonder about the unreachable code.

Maybe I am getting this totally wrong, is this some trick to please a specific compiler or syntax highlighter?